src/main/java/org/apache/commons/text/WordUtils.java

```diff
@@ -739,6 +765,27 @@ private static boolean isDelimiter(final char ch, final char[] delimiters) {
         return false;
     }
 
+  //-----------------------------------------------------------------------
+    /**
+     * Is the codePoint a delimiter.
+     *
+     * @param codePoint the codePint to check
+     * @param delimiters  the delimiters
+     * @return true if it is a delimiter
+     */
+    private static boolean isDelimiter(final int codePoint, final char[] delimiters) {
```

Comment:
Any reason to not make the signature here:
```java
public static String isDelimiter(final int codePoint, final char... delimiters);
```
I suppose we could have a debate about making this public v. private, but I'm not sure I see a reason for it to be exclusively private other than not needing to write as much documentation. I'm open to either way here, it was just something that crossed my mind.